test/specs/contenteditable-textbox.js

```diff
@@ -1,18 +1,26 @@
-const {fixture, matchesKeyboard, matchesMouse} = require('./helpers');
+const {fixture, matchesKeyboard, matchesMouse, FOCUS_RING_STYLE} = require('./helpers');
+const {Key, By} = require('selenium-webdriver');
+const expect = require('expect');
+const driver = global.__driver;
 
 describe('<div role="textbox" contenteditable="true">', function() {
   beforeEach(function() {
-    return fixture('contenteditable-true.html');
+    return fixture('contenteditable-textbox.html');
   });
 
-  // This test seems to fail because FF won't focus a div with contenteditable
-  // if it's the first element on the page. Weird.
-  // https://jsbin.com/cawevo/edit?html,output
-  it.skip('should apply .focus-ring on keyboard focus', function() {
-    return matchesKeyboard();
+  // FF won't focus a div with contenteditable if it's the first element on the page.
```

Comment:
That sounds like it was a lot of fun to debug 🙄

Reply:
Not as much fun as https://github.com/WICG/focus-ring/pull/68/files#diff-6d1daa3464793cab0c4c5d8eeb17e778R178